components/crypto/src/lib.rs

```diff
@@ -22,7 +22,10 @@ pub mod sym_encrypt;
 pub mod test_utils;
 pub mod uid;
 
-#[derive(Clone, Copy, Debug, PartialEq)]
+use derive_more::*;
+
+#[derive(Clone, Copy, Debug, PartialEq, Display)]
+#[display(fmt = "crypto error")]
```

Comment:
Is the `#[display(...)]` thingy a feature of `derive_more`? I have never seen it before, pretty cool!